frontend/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "contexture",
+  "scripts": {
+    "css-compile": "node-sass --include-path node_modules --precision 6 scss -o css/",
+    "css-compile2": "node-sass --include-path node_modules --output-style compressed --source-map true --source-map-contents true --precision 6 scss -o css/",
+    "css-prefix": "postcss --replace css/contexture.css --use autoprefixer --map",
+    "css": "npm-run-all css-compile",
+    "watch": "nodemon -e scss -x \"npm run css\""
+  },
+  "dependencies": {
+    "bootstrap": "next"
+  },
+  "devDependencies": {
+    "autoprefixer": "latest",
```

Comment:
⚠️ `latest` will break the build - use semver range semantics as `^` or `~` instead